src/nwb_datajoint/common/common_task.py

```diff
@@ -13,10 +14,10 @@
 @schema
 class Apparatus(dj.Manual):
     definition = """
-     apparatus_name: varchar(80)
+     apparatus_name: varchar(200)
      ---
      #-> AnalysisNWBFile
```

Comment:
is this commented out? should it be removed?

Reply:
As far as I know this can be removed. I don't remember the original motivation for this, but we don't use it (of course).

Reply:
You're referring to `#-> AnalysisNWBFile` right? I can remove that.

Reply:
I think the whole Apparatus schema can be removed. It formerly referred to the track configuration I think.